build/verify-tools.ts

```diff
@@ -39,7 +39,7 @@ async function verifyTools(): Promise<void> {
 }
 
 const fileCheckRegex = /\w*tools.json/;
-cp.exec('git diff --name-only origin/main -- .', async (error, stdout) => {
+cp.exec('git branch -a', async (error, stdout) => {
```

Comment:
It is not good, this command used to get list of changed files, to track if `tools.json` is changed, but you replase it with list of branches

Reply:
I am testing the build.